core/src/main/java/com/graphhopper/routing/weighting/GenericWeighting.java

```diff
@@ -124,40 +119,16 @@ public double calcWeight(EdgeIteratorState edgeState, boolean reverse, int prevO
         if (time == Long.MAX_VALUE)
             return Double.POSITIVE_INFINITY;
 
-        int spatialId = gEncoder.getSpatialId(edgeState.getFlags());
```

Comment:
This is true, yes. But we could apply speed and access changes when feeding here: https://github.com/boldtrn/graphhopper/blob/59b7e435f11628507cdbc254d27f7ceb3b31501d/core/src/main/java/com/graphhopper/routing/util/DataFlagEncoder.java#L408

I would avoid putting the index in the GenericWeighting ... too many things already there and as we could change speed and accessibility when feeding this is also not necessary for now